codecoverage.sh

```diff
@@ -11,3 +11,4 @@ fi
 lcov --quiet -t 'OOC Test coverage' -o $INFO_FILE -c --directory ./.libs/ooc/ --base-directory .
 genhtml --quiet -o coverage $INFO_FILE --num-spaces 4
 rm $INFO_FILE
+xdg-open 'coverage/index.html'
```

Comment:
This seems to work well. Is there a better, more standardized way to open a webpage? @fredrikbryntesson ?